src/main/java/uk/gov/justice/digital/hmpps/keyworker/controllers/PrisonController.kt

```diff
@@ -90,7 +92,14 @@ class PrisonController(
   fun getStaffDetails(
     @PathVariable prisonCode: String,
     @PathVariable staffId: Long,
-  ): StaffDetails = staffDetails.getFor(prisonCode, staffId)
+  ): StaffDetails = staffDetails.getDetailsFor(prisonCode, staffId)
+
+  @Operation(hidden = true)
+  @GetMapping("/staff/{staffId}/job-classifications")
```

Comment:
Should this URL be added to the excludePathPatterns list in KeyworkerContextConfiguration?